components/company/companyPageCard.tsx

```diff
@@ -42,10 +43,11 @@ const CompanyPageCard = ({ data, schema, index }) => {
   );
 };
 
+const proseClass = "prose-h2:text-3xl/9";
 const CompanyPageCardContent = ({ title, body, schema, index }) => {
   return (
     <article className="col-span-1 size-full rounded border-1 border-gray-300 bg-white px-8 py-4 shadow hover:border-ssw-black dark:border-gray-700 dark:bg-gray-800">
-      <div className="prose prose-h2:text-3xl/9">
+      <div className={classNames("prose", proseClass)}>
         <h2
           className="my-1"
```

Comment:
Just leave this the way it was, it's better to use the whitelist to keep track of which classes are exempt from the linting check

```suggestion
      <div className="prose prose-h2:text-3xl/9">
        <h2
          className="my-1"
```

Reply:
Done